ledger/src/snapshot_utils.rs

```diff
@@ -270,10 +287,19 @@ where
         .ok_or_else(|| get_io_error("No snapshots found in snapshots directory"))?;
 
     // Rebuild the root bank
-    info!("Loading from {:?}", &root_paths.snapshot_file_path);
+    info!("Loading bank from {:?}", &root_paths.snapshot_file_path);
     let file = File::open(&root_paths.snapshot_file_path)?;
     let mut stream = BufReader::new(file);
-    let bank: Bank = deserialize_from(&mut stream)?;
+    let bank: Bank = match snapshot_version {
+        env!("CARGO_PKG_VERSION") => deserialize_from(&mut stream)?,
+        "v0.22.3" => {
+            let bank0223: solana_runtime::bank::LegacyBank0223 = deserialize_from(&mut stream)?;
```

Comment:
nits: Maybe prefixed (`solana_runtime::bank::LegacyBank0223`) with a good reason as a some eye catch? :)

Reply:
I wanted to keep the scope of LegacyBank0223 usage as narrow as possible, as opposed to touching the global `use` at the top of the file -- makes removal of LegacyBank0223 in the future easier (and there's only one reference in to LegacyBank0223 in this file anyway)